chunky/src/java/se/llbit/chunky/world/biome/Biome.java

```diff
@@ -0,0 +1,73 @@
+/* Copyright (c) 2013 Jesper Öqvist <jesper@llbit.se>
+ *
+ * This file is part of Chunky.
+ *
+ * Chunky is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Chunky is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with Chunky.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package se.llbit.chunky.world.biome;
+
+import org.apache.commons.math3.util.FastMath;
+import se.llbit.chunky.renderer.scene.Scene;
+import se.llbit.math.ColorUtil;
+
+/**
+ * @author Jesper Öqvist <jesper@llbit.se>
+ */
+public class Biome {
+  public final String resourceLocation;
+  public final String name;
+  public final float temp;
```

Comment:
Took me a moment to get what `temp` meant in this context…

```suggestion
  public final float temperature;
```